crs_application/src/task_managers/scan_acquisition_manager.cpp

```diff
@@ -48,6 +48,7 @@ static const std::string FREESPACE_MOTION_PLAN_SERVICE = "plan_freespace_motion"
 static const std::string MANAGER_NAME = "ScanAcquisitionManager";
 static const std::string SCAN_POSES_TOPIC = "scan_poses";
 static const std::string DEFAULT_WORLD_FRAME_ID = "world";
+static const std::string CONTROLLER_CHANGER_SERVICE = "test_serv";
```

Comment:
I'd prefer a more descriptive name

Reply:
I changed it to be more descriptive, but missed changing the reference here. Updated now